File: Hervard CS50/Learning C/number_guessing.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>

int main(){
    int guess;
    int random_number;
    int attemp = 0;
    int max_attemp = 10;

    srand(time(NULL));

    random_number = (rand()%100)+1;

    printf("Welcome to number guessing game \n");
    printf("You have 10 attemps to find the number \n");

    while (attemp < max_attemp)
    {
        printf("Attemp %d : Enter your number = ", attemp +1);
        scanf("%d", &guess);
        attemp ++;

        if (random_number == guess){
            printf("You Guessed correct number. \n");
            printf("Your guessed number is %d \n", guess);

            return 0;
        }
        else if(random_number < guess){
            printf("Too high, guess again.\n");
        }
        else{
            printf("Too low, guess again.\n");
        }

    }
    printf("You have ran out of attemps. The number was %d", random_number);
        return 0;

}

File: Hervard CS50/Python/number_guessing.py
import random


guess = 0
attemps = 0
max_attemps = 10
random_number = random.randint(1,100)


print("Welcome to Number guessing game")
print("You have", max_attemps,"attempts. Select a number from 1 to 100")

while(attemps < max_attemps):
    try:
        guess = int(input("Enter your number >> "))
    except ValueError:
        print("Please a vaild number")
        continue
    attemps += 1

    if (guess == random_number):
        print("You have choosen the correct number")
        print("Your number is ", random_number)

    elif(guess < random_number):
        print("You guess is too low. Try again")
    else:
        print("Your guess is too high try again")



if (attemps == max_attemps):
    print("You are out of guess. The number was ", random_number)
    print("Game over")

    

